### PR TITLE
[DC-3126] Update obsolete comments about backfill in `convert_pre_post_coordinated_concepts.py`

### PR DESCRIPTION
**Pull request by hiro-mishima**

### CODE REVIEW

File: data_steward/cdr_cleaner/cleaning_rules/convert_pre_post_coordinated_concepts.py [hunk 1]
```diff
@@ -170,10 +170,7 @@ def get_query_specs(self, *args, **keyword_args) -> query_spec_list:
         observation_concept_ids and value_as_concept_ids.
         New observation_id is generated as:
             100,000,000,000 * ((n)th record from the mapping table for the value_source_concept_id) + original observation_id.
-
-        Note: "backfill_pmi_skip_codes" has a similar logic. It generates new observation_ids with 1,000,000,000,000.
-              This CR generates the IDs with 100,000,000,000 - 400,000,000,000. So, the IDs do not overlap with the ones from
-              "backfill_pmi_skip_codes".
+            "n" can be 1 - 4. So, the newly generated IDs will be in the range of 100,000,000,000 - 499,999,999,999.
```
Comment (lrwb-aou) [inline data_steward/cdr_cleaner/cleaning_rules/convert_pre_post_coordinated_concepts.py:173]: @hiro-mishima This makes sense.  Nice update.  Would it be worth it to put a similar comment in the jinja template too?

**Review by lrwb-aou — APPROVED** [review]
This makes sense.  Nice comment.

File: data_steward/cdr_cleaner/cleaning_rules/convert_pre_post_coordinated_concepts.py (hunk 1 above)
Comment (hiro-mishima) [reply data_steward/cdr_cleaner/cleaning_rules/convert_pre_post_coordinated_concepts.py:175]: Great idea! I added the comment to the JINJA template.

File: data_steward/cdr_cleaner/cleaning_rules/convert_pre_post_coordinated_concepts.py [hunk 2]
```diff
@@ -90,6 +90,8 @@
 INSERT INTO `{{project}}.{{dataset}}.observation`
 ({{observation_fields}})
 SELECT 
+    -- ROW_NUMBER() here can be 1 - 4. So, the newly generated IDs will be --
+    -- in the range of 100,000,000,000 - 499,999,999,999. --
```
Comment (lrwb-aou) [inline data_steward/cdr_cleaner/cleaning_rules/convert_pre_post_coordinated_concepts.py:94]: 👍

**Pull request merged**